components/Blockquote/src/index.css

```diff
@@ -0,0 +1,28 @@
+.denhaag-blockquote {
+  border-inline-start: 4px solid var(--denhaag-color-green-1, #d5e7d4);
```

Comment:
Throughout the library we avoid using tokens from the `Proprietary` package in CSS. Instead, tokens used in CSS should usually be component specific. `--denhaag-blockquote-border-color`. The CSS should by default contain no proprietary styles, as another organisation should be able to use it and have to make their own design decisions, rather than using the Den Haag style as default.

Reply:
Additional to @bddjong comment, i would suggest to split this onto three CSS variables, otherwise we're making a design choice for the person/client using this component.

My suggestions are:
1. `--denhaag-blockquote-border-color`
2. `--denhaag-blockquote-border-width`
3. `--denhaag-blockquote-border-style`